### DISCUSSION TITLE
Navigation back gesture

### DISCUSSION THREAD

**Discussion by erquhart** [gorhom/react-native-bottom-sheet#1391] [Q&A]
I'm using a stack navigation inside of bottom sheet, but it seems like bottom sheet is blocking the swipe-from-edge gesture to go back to the last view in the stack. Is there a way to have bottom sheet ignore side or edge gestures?

**Comment by jokerhp6789** [comment]
Hi, I encounter   the same problem  as you describe , did you find any solutions for this yet ?

**Comment by JestemTrzcinska** [comment]
same problem here, any ideas?

**Comment by idlework** [comment]
You can use `NativeViewGestureHandler` from the `react-native-gesture-handler` package to avoid interruptions.

```
<NativeViewGestureHandler disallowInterruption={true}>
  {children}
</NativeViewGestureHandler>
```
